invoice.py

```diff
@@ -454,12 +454,15 @@ def _sign_facturae(self, xml_string, certificate_password):
 
         def _sign_file(cert, password, request):
             # get key and certificates from PCK12 file
-            (
-                private_key,
-                certificate,
-                additional_certificates,
-                ) = pkcs12.load_key_and_certificates(cert, password)
-
+            try:
+                (
+                    private_key,
+                    certificate,
+                    additional_certificates,
+                    ) = pkcs12.load_key_and_certificates(cert, password)
+            except ValueError:
+                raise UserError(gettext(
+                    'account_invoice_facturae.msg_incorrect_password'))
```

Comment:
At the moment, except return:

```
/cryptography/hazmat/backends/openssl/backend.py
Invalid password or PKCS12 data
```

but the error could it from other reasons besides the password.

I thing we need to capture the error message and not a fixed message.

Reply:
Changed to the error message: https://github.com/NaN-tic/trytond-account_invoice_facturae/pull/7/commits/42067cbca36e90e8c825b4cbffa0a875ee7613e0